data/strings/strings.txt

```diff
@@ -22266,4 +22318,4 @@
     uk = Маршрут
     vi = Dấu chân
     zh-Hans = 追踪
-    zh-Hant = 追踪
+    zh-Hant = 追踪    
```

Comment:
?

Reply:
FYI we can ignore the whitespace, it will be re-formatted by generate_localizations.sh (strings_utils.py) anyway